.github/workflows/auto_tag.yml

```diff
@@ -0,0 +1,140 @@
+name: Create Tag
+
+on:
+  workflow_dispatch:
```

Comment:
Does this mean this workflow is triggered manually only? TL;DR

Reply:
Right now it's only a manual trigger as @webhdx has some worries regarding a fully automatic run.

Reply:
> Right now it's only a manual trigger as @webhdx has some worries regarding a fully automatic run.

Yeah, that's why I asked. Manual is the safest choice here. Nice :+1: